backend/spotihue/tasks.py

```diff
@@ -11,21 +11,65 @@
 logger = logging.getLogger(__name__)
 
 
+def is_spotihue_running() -> bool:
+    """Queries for the SpotiHue task by its cached spotihue_task_id. There should only be one
+    run_spotihue task running at any given time.
+
+    Returns:
+         bool: Whether the SpotiHue (run_spotihue) task is running.
```

Comment:
Is this description confusing? So it's just looking for the cached task ID. However, if the task fails in some way the technically it would not be running anymore, but the cached task ID would still exist.

Reply:
I try to address this with the clear_spotihue_task_id task...but yeah you're right

Reply:
Maybe we just change the doc string here to reflect that?

Reply:
Yeah sure! But wait, what about the docstring do we want to change? I'm not sure how to improve the wording here, but if you have something in mind I'm happy to update it.